cmd/inspr/cli/get_brokers.go

```diff
@@ -0,0 +1,28 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/inspr/inspr/pkg/cmd"
+	cliutils "github.com/inspr/inspr/pkg/cmd/utils"
+)
+
+var getBorkers = cmd.NewCmd("brokers").
+	WithDescription("Retreives brokers curently installed on cluster").
+	NoArgs(
+		func(ctx context.Context) error {
+			client := cliutils.GetCliClient()
+			out := cliutils.GetCliOutput()
+			resp, err := client.Brokers().Get(context.Background())
+			if err != nil {
+				cliutils.RequestErrorMessage(err, out)
+				return err
+			}
+
+			fmt.Fprintf(out, "DEFAULT:\n%s\n", resp.Default)
+			fmt.Fprintln(out, "AVAILIBLE:")
+			lines := []string(resp.Installed)
+			printTab(&lines)
+			return nil
+		})
```

Comment:
Why not join this in the same file as the clusterCommand file?

Taking the deleteCmd for example, you have two subcommands for the `cluster` command that are similar in size and related to a same command. That way it might be interesting to have them together in the same file

Reply:
I don't know if I understood exactly what you said, but I think it's interesting to create a "clusterCommand" file or similar, for it seems a bit odd to have the `inspr cluster` "main" file being one called `auth_init.go`

Reply:
I created a tech P. to deal with the integration of these 3 files. These commands also apparently require testing